.github/workflows/rust.yml

```diff
@@ -334,6 +334,8 @@ jobs:
         run: |
           export PATH=$PATH:$HOME/d/protoc/bin
           cargo test --lib --tests --bins --features avro,json,backtrace
+          # 8MB stack size otherwise the cli_integration test fails  
```

Comment:
I added this to workaround the issue @tisonkun  reports https://github.com/apache/datafusion/pull/10392#issuecomment-2143357574

If it passes CI I will file a ticket to track

Reply:
Seems failure retains :<

Reply:
👍  reverting the change and will try something else